ddtrace/internal/constants.py

```diff
@@ -0,0 +1,24 @@
+# Publicly-used constants
+ANALYTICS_SAMPLE_RATE_KEY = "_dd1.sr.eausr"
+MANUAL_DROP_KEY = "manual.drop"
+MANUAL_KEEP_KEY = "manual.keep"
+SPAN_KIND = "span.kind"
+SPAN_MEASURED_KEY = "_dd.measured"
```

Comment:
This is definitely used by customers, used by use internally

Reply:
Yep these ones are all public. I have adopted @majorgreys strategy of internalising all constants and exporting only the public ones.

Reply:
sorry, I must have read over that

Reply:
Just curious, if we export only the public ones isn't that technically a breaking change for customers currently using the non-public constants?

Reply:
correct